APPS-1858 TreeTurnaroundTime threshold

Gvaihir opened the pull request:
Changes needed for dxCompiler

Gvaihir committed:
APPS-1858_TAT_threshold
(feat) DxObject - added new TreeTurnaroundTimeThreshold Field

Gvaihir committed:
Merge branch 'develop' into APPS-1858_TAT_threshold

Gvaihir committed:
APPS-1858_TAT_threshold
(feat) DxObject - added new TreeTurnaroundTimeThreshold Field

commandlinegirl commented on api/src/main/scala/dx/api/DxApplet.scala:
```diff
@@ -22,24 +22,26 @@ case class DxAppletDescribe(project: String,
                             ignoreReuse: Option[Boolean] = None,
-                            hidden: Option[Boolean] = None)
+                            hidden: Option[Boolean] = None,
+                            treeTurnaroundTimeThreshold: Option[Long] = None)
```
Can you add a docstring or comment on what this input is?

commandlinegirl replied:
Here and in other places where needed.

Gvaihir replied:
Added in [2072d6d](https://github.com/dnanexus/dxScala/pull/123/commits/2072d6debfbbfbc12d29e76767905406f4145daf)

commandlinegirl reviewed (dismissed):
LGTM modulo a minor request

Gvaihir committed:
APPS-1858_TAT_threshold
(feat) DxAppDescribe, DxAppletDescribe, DxWorkflowDescribe - added a docstring for TreeTurnaroundTimeThreshold
